Anlab.Mvc/Services/IOrderService.cs

```diff
@@ -25,13 +28,15 @@ public class OrderService : IOrderService
     {
         private readonly ApplicationDbContext _context;
         private readonly ITestItemPriceService _itemPriceService;
+        private readonly ConnectionSettings _connectionSettings;
         private readonly AppSettings _appSettings;
 
-        public OrderService(ApplicationDbContext context, ITestItemPriceService itemPriceService, IOptions<AppSettings> appSettings)
+        public OrderService(ApplicationDbContext context, ITestItemPriceService itemPriceService, IOptions<AppSettings> appSettings, IOptions<ConnectionSettings> connectionSettings)
         {
             _context = context;
-            _itemPriceService = itemPriceService;
+            _itemPriceService = itemPriceService;            
             _appSettings = appSettings.Value;
+            _connectionSettings = connectionSettings.Value; //Will need to push to Anlab.

```

Comment:
@srkirkland I'm assuming that when we push data to anlab, we will need the connection then. I can remove it until we actually do that if you want.